test/puppetlabs/trapperkeeper/authorization/acl_test.clj

```diff
@@ -123,6 +123,11 @@
     (testing "should allow anything"
       (is (acl/allowed? acl {:certname "anything" :extensions {}})))))
 
+(deftest test-rbac-permission
+  (let [acl (acl/allow {:rbac {:permission "whee"}} )]
+                         (testing "RBAC permission string matching"
+                           (is (acl/allowed? acl { :permission "whee"} ) ))))
```

Comment:
Why would it be called this way?

Reply:
We discussed this and decided this test is outside the scope of this change, because it involves the spot in the code where we will actually need to do the RBAC validation. This test will be reworked to test the right thing in a later PR. For now, we've removed it.